From bcaac964d8e6fa5a2011895c8d7e32e7c114f420 Mon Sep 17 00:00:00 2001
From: pmandri <padmashree.mandri@intel.com>
Date: Wed, 25 Aug 2021 16:04:39 +0530
Subject: [PATCH] Set Audio Volume to Maximum

Set Audio volume for STREAM_MUSIC to maximum
Tracked-On: OAM-98036
Signed-off-by: pmandri <padmashree.mandri@intel.com>
---
 services/core/java/com/android/server/audio/AudioService.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/audio/AudioService.java b/services/core/java/com/android/server/audio/AudioService.java
index fb9cf0dc5f06..785995299959 100755
--- a/services/core/java/com/android/server/audio/AudioService.java
+++ b/services/core/java/com/android/server/audio/AudioService.java
@@ -754,7 +754,7 @@ public class AudioService extends IAudioService.Stub
                         MAX_STREAM_VOLUME[AudioSystem.STREAM_MUSIC] / 4;
             } else {
                 AudioSystem.DEFAULT_STREAM_VOLUME[AudioSystem.STREAM_MUSIC] =
-                        MAX_STREAM_VOLUME[AudioSystem.STREAM_MUSIC] / 3;
+                        MAX_STREAM_VOLUME[AudioSystem.STREAM_MUSIC] ;
             }
         }
 
-- 
2.17.1

